### PR TITLE
update documentation for SplitOrMergeObjects

### PR DESCRIPTION
**Pull request by karhohs**

### CODE REVIEW

File: cellprofiler/modules/splitormergeobjects.py [hunk 1]
```diff
@@ -82,147 +85,156 @@ class SplitOrMergeObjects(cpm.Module):
...
-            polygon (the convex hull) that encloses all of a parent's child
-            objects. The convex hull will be truncated to include only those
-            pixels in the parent - in that case it may not truly be convex.
-            Choose %(UM_DISCONNECTED)s to leave the children as
-            disconnected pieces. Choose %(UM_CONVEX_HULL)s to create an
-            output object that is the convex hull around them all.
-            """ % globals()
+                doc="""\
+*(Used only with the %(UNIFY_PARENT)s unification method)*
+**SplitOrMergeObjects** can either unify the child objects and keep them
```
Comment (mcquin) [inline cellprofiler/modules/splitormergeobjects.py]: Add a blank link between `*(Used only with...)*` and the next line.

File: cellprofiler/modules/splitormergeobjects.py [hunk 2]
```diff
@@ -82,147 +85,156 @@ class SplitOrMergeObjects(cpm.Module):
...
-            and keep them disconnected or it can find the smallest convex
-            polygon (the convex hull) that encloses all of a parent's child
-            objects. The convex hull will be truncated to include only those
-            pixels in the parent - in that case it may not truly be convex.
-            Choose %(UM_DISCONNECTED)s to leave the children as
-            disconnected pieces. Choose %(UM_CONVEX_HULL)s to create an
-            output object that is the convex hull around them all.
-            """ % globals()
+                doc="""\
+*(Used only with the %(UNIFY_PARENT)s unification method)*
```
Comment (mcquin) [inline cellprofiler/modules/splitormergeobjects.py]: Add quotes around setting text:

```
*(Used only with the "%(UNIFY_PARENT)s" unification method)*
```

File: cellprofiler/modules/splitormergeobjects.py [hunk 3]
```diff
@@ -82,147 +85,156 @@ class SplitOrMergeObjects(cpm.Module):
...
-            you enter here, in pixels, will be unified. If you choose zero
-            (the default), only objects that are touching will be unified.
-            Note that <i>%(OPTION_UNIFY)s </i> will not actually connect or bridge
-            the two objects by adding any new pixels; it simply assigns the same object number
-            to the portions of the object. The new, unified object
-            may therefore consist of two or more unconnected components.""" % globals())
+                0, minval=0, doc="""\
+*(Used only with the %(OPTION_UNIFY)s option and the
+%(UNIFY_DISTANCE)s method)*
+Objects that are less than or equal to the distance you enter here, in
```
Comment (mcquin) [inline cellprofiler/modules/splitormergeobjects.py]: Add a new line between `*(Used only with...)*` and the next line.

File: cellprofiler/modules/splitormergeobjects.py [hunk 4]
```diff
@@ -82,147 +85,156 @@ class SplitOrMergeObjects(cpm.Module):
...
-            <i>(Used only with the %(OPTION_UNIFY)s option and the %(UNIFY_DISTANCE)s method)</i><br>
-            Objects that are less than or equal to the distance
-            you enter here, in pixels, will be unified. If you choose zero
-            (the default), only objects that are touching will be unified.
-            Note that <i>%(OPTION_UNIFY)s </i> will not actually connect or bridge
-            the two objects by adding any new pixels; it simply assigns the same object number
-            to the portions of the object. The new, unified object
-            may therefore consist of two or more unconnected components.""" % globals())
+                0, minval=0, doc="""\
+*(Used only with the %(OPTION_UNIFY)s option and the
```
Comment (mcquin) [inline cellprofiler/modules/splitormergeobjects.py]: Add quotes around module setting values:

```python
*(Used only with the "%(OPTION_UNIFY)s" option and the
"%(UNIFY_DISTANCE)s" method)*
```

File: cellprofiler/modules/splitormergeobjects.py [hunk 5]
```diff
@@ -82,147 +85,156 @@ class SplitOrMergeObjects(cpm.Module):
...
         self.wants_image = cps.Binary(
-                "Unify using a grayscale image?", False, doc="""
-            <i>(Used only with the %(OPTION_UNIFY)s option)</i><br>
-            Select <i>%(YES)s</i> to use the objects' intensity features to determine whether two
-            objects should be unified. If you choose to use a grayscale image,
-            <i>%(OPTION_UNIFY)s</i> will unify two objects only if they
-            are within the distance you have specified <i>and</i> certain criteria about the objects
-            within the grayscale image are met.""" % globals())
+                "Unify using a grayscale image?", False, doc="""\
+*(Used only with the %(OPTION_UNIFY)s option)*
```
Comment (mcquin) [inline cellprofiler/modules/splitormergeobjects.py]: Add quotes around setting values:

```python
*(Used only with the "%(OPTION_UNIFY)s" option)*
```

File: cellprofiler/modules/splitormergeobjects.py [hunk 6]
```diff
@@ -82,147 +85,156 @@ class SplitOrMergeObjects(cpm.Module):
...
         self.wants_outlines = cps.Binary(
-                "Retain outlines of the relabeled objects?", False, doc="""
-            %(RETAINING_OUTLINES_HELP)s""" % globals())
+                "Retain outlines of the relabeled objects?", False, doc="""%(RETAINING_OUTLINES_HELP)s""" % globals())
 
         self.outlines_name = cps.OutlineNameProvider(
                 'Name the outlines',
-                'RelabeledNucleiOutlines', doc="""
-            %(NAMING_OUTLINES_HELP)s""" % globals())
+                'RelabeledNucleiOutlines', doc="""%(NAMING_OUTLINES_HELP)s""" % globals())
```
Comment (mcquin) [inline cellprofiler/modules/splitormergeobjects.py]: Unnecessary quotes. `doc=NAMING_OUTLINES_HELP` is sufficient.

File: cellprofiler/modules/splitormergeobjects.py [hunk 7]
```diff
@@ -1,7 +1,10 @@
 # coding=utf-8
 
 """
-**Split or merge** previously identified objects.
+SplitOrMergeObjects
+===================
+
+**SplitOrMergeObjects** previously identified objects.
```
Comment (bethac07) [inline cellprofiler/modules/splitormergeobjects.py]: Can you make this a more complete sentence? It's currently a bit ambiguous.

**Pull request merged**